ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/index.tsx

```diff
@@ -75,7 +75,7 @@ const RunComponent = () => {
       const url = window.URL.createObjectURL(blob);
       const a = document.createElement('a');
       a.href = url;
-      a.download = 'logs.zip';
+      a.download = `logs-global-run-${runId}.zip`;
```

Comment:
Hmm. Apparently the backend already provides a proper name here:

https://github.com/eclipse-apoapsis/ort-server/blob/03f5738d78aacb1ef4814b3d3eb657cf641cbee8/core/src/main/kotlin/api/RunsRoute.kt#L98

So, wouldn't it be possible to use that name from the response?

Reply:
See https://github.com/eclipse-apoapsis/ort-server/pull/867.